PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/CameraScript/TransparencyController.rbxmx

```diff
@@ -137,7 +137,13 @@ local function CreateTransparencyController()
 	end
 
 	function module:SetSubject(subject)
-		local character = subject and subject:IsA('Humanoid') and subject.Parent
+		local character = nil
+		if subject and subject:IsA("Humanoid") then
+			character = subject.Parent
+		end
+		if subject and subject:IsA("VehicleSeat") then
+			character = subject.Occupant.Parent
```

Comment:
I didn't think of it; it's highly unlikely to have a VehicleSeat as a camera subject without being in the seat, but not impossible... will fix.